docs/src/main/sphinx/connector/hive.rst

```diff
@@ -388,6 +388,11 @@ Property Name                                      Description
 ``hive.query-partition-filter-required``           Set to ``true`` to force a query to use a partition filter.   ``false``
                                                    You can use the ``query_partition_filter_required`` catalog
                                                    session property for temporary, catalog specific use.
+
+``hive.statistics_enabled``                        Disable :doc:`/optimizer/statistics`, and therefore           ``true``
```

Comment:
it is confusing that the name of this is *enabled, but the description starts out with "Disable..." but the default is true.